browser/src/map/handler/Map.SlideShow.js

```diff
@@ -115,6 +115,12 @@ L.Map.SlideShow = L.Handler.extend({
 		var separator = (this._slideURL.indexOf('?') === -1) ? '?' : '&';
 		this._slideShow.src = this._slideURL + separator + 'StartSlideNumber=' + this._startSlideNumber;
 		this._slideShow.contentWindow.focus();
+
+		var that = this;
+		this._slideShow.addEventListener('load', function() {
+			var videos = that._slideShow.contentWindow.document.getElementsByTagName('video');
+			L.SVGGroup.prototype.checkVideoSupport(undefined, undefined, videos);
```

Comment:
Sorry, I didn't mean `checkVideoSupport` should handle the `videos` case.

I mean that for `videos` we do exactly the same thing as for `source`, so we should call the `checkVideoSupport` for each video entry.

We want to reduce code duplication, not put everything in one function.

Since the `source` tag is a child of the `video` tag, we should be able to have `checkVideoSupport` that takes a `video` node and add the listener for the `video`, query the `source` node, add the listener there too, if it exists.

_Another_ function can then be used to iterate over all videos and call `checkVideoSupport` on each `video` node.

(By the way, 'check' in 'checkVideoSupport' implies the function returns the results of the check. This is misleading, since we don't do that. Instead, we should call it 'addVideoSupportHandlers' or something more accurate.)